### PR TITLE
remove reg + delete draft

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#9195

*Description of changes:*
- see ticket for details

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/views/Dashboard.vue [hunk 1]
```diff
@@ -353,6 +408,59 @@ export default class Dashboard extends Vue {
     this.loading = false
   }
 
+  private myRegActionDialogHandler (proceed: boolean): void {
```
Comment (jordiwes) [inline ppr-ui/src/views/Dashboard.vue:411]: can you pop a little comment on what this is for? It's not totally obvious to me :)

**Review by jordiwes — APPROVED** [review]
Looks good, just one small comment

File: ppr-ui/src/views/Dashboard.vue (hunk 1 above)
Comment (kialj876) [reply ppr-ui/src/views/Dashboard.vue:411]: Yes I will! I will add it in my next PR (it is just waiting for this one to go through)

**Pull request merged**